server/api/users/users.test.ts

```diff
@@ -72,4 +72,96 @@ describe('auth router', () => {
       });
     });
   });
+
+  describe('PATCH /:id', () => {
+    it('returns 403 if profile id is not equal to logged in user id', async () => {
+      const res = await server.exec.patch('/api/users/54321');
+      expect(res.status).toBe(403);
+    });
+
+    it('returns 404 if specified user doesn\'t exist', async () => {
+      const res = await server.exec.patch('/api/users/1');
+      expect(res.status).toBe(404);
+    });
+
+    it('updates the specified user', async () => {
+      const user = await User.create({
+        email: randEmail(),
+        discord_user_id: randNumber().toString(),
+        discord_name: `${randUserName()}#${randNumber()}`,
+        bio: 'I like to put cat pictures on Caleb\'s desktop & my owner owns nerdwallet',
+        twitter_username: randUserName(),
+        linkedin_url: `https://www.linkedin.com/in/${randUserName()}/`,
+        github_username: randUserName(),
+        website: 'https://leonnoel.com/',
+      });
+
+      server.login(user);
+
+      const fieldsToUpdate = {
+        bio: 'I was updated.',
+        twitter_username: 'Updated@updated',
+        linkedin_url: 'updated',
+        github_username: 'gotUpdated',
+        website: 'https://thiswasupdated.com/',
+      };
+
+      const res = await server.exec.patch(`/api/users/${server.loggedInUser.dataValues.id}`).send(fieldsToUpdate);
```

Comment:
instead of `server.loggedInUser.dataValues.id` i think you can just do `user.id` here